src/views/Send/Send.vue

```diff
@@ -381,10 +381,7 @@ export default {
       return BN(this.amount).plus(BN(this.currentFee))
     },
     isEIP1559Fees() {
-      return (
-        this.assetChain === 'ETH' ||
-        (this.assetChain === 'MATIC' && this.activeNetwork === 'testnet')
-      )
+      return this.assetChain === 'ETH' || this.assetChain === 'MATIC'
```

Comment:
cryptoassets[this.asset].chain === ChainId.Ethereum is probably the correct way to do this
but it is the same result

Reply:
just replaced it